src/MetricExporter.ts

```diff
@@ -123,11 +125,13 @@ export class DynatraceMetricExporter implements PushMetricExporter {
 						lines = lines.concat(this.serializeCounter(metric));
 						break;
 					case InstrumentType.OBSERVABLE_GAUGE:
+						lines = lines.concat(this.serializeGauge(metric));
+						break;
 					case InstrumentType.UP_DOWN_COUNTER:
-						lines.push(...this.serializeGauge(metric));
+						lines = lines.concat(this.serializeUpDownCounter(metric));
```

Comment:
Oh :sweat_smile: 
Thanks for updating these as well. Did not comment on every instance when I reviewed the other PR. :sweat_smile: